.github/workflows/nodepackage.yml

```diff
@@ -0,0 +1,35 @@
+name: Node.js package
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
shouldnt version be pinned too ?

Reply:
Really shouldn't matter either way. I can pin if you'd prefer.

Reply:
not a big deal, it's not something to be used by user so let's merge